BookLibrary-cdi/src/main/java/org/booklibrary/app/manager/beans/AuthorManager.java

```diff
@@ -26,72 +32,80 @@
+            throw new EntityManagerException(errorMsg, e);
         }
-        LOG.debug("Persist entity: {}", entity);
+        logger.debug("Persist entity: {}", entity);
         return entity;
     }
 
-    @Override
-    public Author update(Author obj) throws EntityPersistenceException {
+    public Author update(Author obj) throws EntityManagerException {
+        logger.debug("update invoked for obj: {}", obj);
         Author updated;
         try {
             updated = authorHome.update(obj);
         } catch (Exception e) {
             String errorMsg = "Failed to update object: {}";
-            LOG.error(errorMsg, obj);
-            throw new EntityPersistenceException(errorMsg, e);
+            logger.error(errorMsg, obj);
+            throw new EntityManagerException(errorMsg, e);
         }
-        LOG.debug("Update entity: {}", updated);
+        logger.debug("Update entity: {}", updated);
         return updated;
     }
 
-    @Override
-    public void remove(EntityIdentifier key) throws EntityPersistenceException {
+    public void removeByPk(EntityIdentifier key) throws EntityManagerException {
+        logger.debug("Remove called for entity with key: {}", key);
         try {
-            authorHome.remove(key);
+            authorHome.removeByPk(key);
         } catch (Exception e) {
-            String errorMsg = "Failed to remove object with pk: {}";
-            LOG.error(errorMsg, key);
-            throw new EntityPersistenceException(errorMsg, e);
+            String errorMsg = "Failed to remove object with key: {}";
+            logger.error(errorMsg, key);
+            throw new EntityManagerException(errorMsg, e);
         }
-        LOG.debug("Remove entity with pk: {}", key);
     }
 
-    @Override
-    public void remove(String uuid) throws EntityPersistenceException{
-
+    public void removeByUuid(String uuid) throws EntityManagerException {
+        removeByPk(new EntityIdentifier(uuid));
     }
 
-    @Override
-    public void removeAll() throws EntityPersistenceException{
-
+    public void removeAll() throws EntityManagerException {
+        logger.debug("Remove all invoked");
+        try {
+            authorHome.removeAll();
+        } catch (Exception e) {
+            String errorMsg = "Failed to remove all objects ";
+            logger.error(errorMsg);
+            throw new EntityManagerException(errorMsg, e);
+        }
     }
 
-    @Override
     public Author findByPk(EntityIdentifier key) {
-        return null;
+        logger.debug("Find invoked for object with key: {}", key);
+        return authorFacade.findByPk(key);
     }
 
-    @Override
     public Author findByUuid(String uuid) {
-
+        logger.debug("Find invoked for object with key: {}", uuid);
         return authorFacade.findByUuid(uuid);
     }
 
-    @Override
     public List<Author> findAll() {
+        logger.debug("Find all called");
+        List<Author> authors =  authorFacade.findAll();
```

Comment:
there is no need in creating local variable here, just return authorFacade.findAll()